Measurements/Measurements.ipynb

```diff
@@ -572,6 +572,43 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Solution:**\n",
```

Comment:
I don't think we should offer the solution to the task before we give people a chance to attack the problem themselves... The Measurements workbook is [still in progress](https://github.com/microsoft/QuantumKatas/pull/277) and possibly abandoned, so I won't suggest you to wait until it's ready :-) 

I would suggest to create a new file `Workbook_Measurements_Part2.ipynb` and to put this solution there, similar to how we did it with [Superposition workbook](https://github.com/microsoft/QuantumKatas/blob/master/Superposition/Workbook_Superposition_Part2.ipynb) - the second part will mention that the first part is in progress, and will be able to describe the solution to 2.2 completely independently.

Reply:
ok, I did this and also described the other tasks 2.2 and 2.3 in the workbook using the comments from the reference implementation. I hope you agree.